src/Keycloak.Net.Core/Organizations/KeycloakClient.cs

```diff
@@ -0,0 +1,216 @@
+							   .GetJsonAsync<long>(cancellationToken: cancellationToken)
+							   .ConfigureAwait(false);
+
+	public async Task<List<Member>> GetOrganizationMembersAsync(string realm,
+																string organizationId,
+																bool? exact = null,
+																int? first = null,
+																int? max = null,
+																string? search = null,
+																CancellationToken cancellationToken = default)
+	{
+		var queryParams = new Dictionary<string, object?>
+		{
+			[nameof(exact)] = exact,
+			[nameof(first)] = first,
+			[nameof(max)] = max,
+			[nameof(search)] = search
+		};
+		var response = await GetBaseUrl(realm).AppendPathSegment($"/admin/realms/{realm}/organizations/{organizationId}/members")
+											  .SetQueryParams(queryParams)
+											  .GetJsonAsync<List<Member>>(cancellationToken: cancellationToken)
+											  .ConfigureAwait(false);
+		return response;
+
+	}
+
+	public async Task<Member> GetOrganizationMemberAsync(string realm,
+														 string organizationId,
+														 string memberId,
+														 CancellationToken cancellationToken = default) =>
+		await GetBaseUrl(realm).AppendPathSegment($"/admin/realms/{realm}/organizations/{organizationId}/members/{memberId}")
+							   .GetJsonAsync<Member>(cancellationToken: cancellationToken)
+							   .ConfigureAwait(false);
+
+	public async Task<bool> DeleteMemberFromOrganizationAsync(string realm,
+															  string organizationId,
+															  string memberId,
+															  CancellationToken cancellationToken = default)
+	{
+		var response = await GetBaseUrl(realm).AppendPathSegment($"/admin/realms/{realm}/organizations/{organizationId}/members/{memberId}")
+											  .DeleteAsync(cancellationToken: cancellationToken)
+											  .ConfigureAwait(false);
+
+		return response.ResponseMessage.IsSuccessStatusCode;
+	}
+
+	
+
+	public async Task<List<Organization>> GetOrganizationsForMemberAsync(string realm,
+																		 string organizationId,
+																		 string memberId,
+																		 CancellationToken cancellationToken = default)
+	{
+		return await GetBaseUrl(realm).AppendPathSegment($"/admin/realms/{realm}/organizations/{organizationId}/members/{memberId}/organizations")
+									  .GetJsonAsync<List<Organization>>(cancellationToken: cancellationToken)
+									  .ConfigureAwait(false);
+	}
+
+	public async Task<bool> AddMemberToOrganizationAsync(string realm,
+														 string organizationId,
+														 string userId,
+														 CancellationToken cancellationToken = default)
+	{
+		var response = await GetBaseUrl(realm).AppendPathSegment($"/admin/realms/{realm}/organizations/{organizationId}/members")
+											  .PostJsonAsync(userId, cancellationToken: cancellationToken)
+											  .ConfigureAwait(false);
+		return response.ResponseMessage.IsSuccessStatusCode;
+	}
+
+	public async Task<bool> InviteUserToOrganizationAsync(string realm,
```

Comment:
I considered this originally, but they both have the ability to invite existing users, specifically, if the email address provided belongs to an existing user it will send them an invite to join, and if not it will send them an invite to register.

I am going to add XML comments to explain the difference.